### PR TITLE
Integration tests for Blob Service

### PR DESCRIPTION
**Pull request by KateYatsiuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @ita-social-projects/lv-734-02-streetcode 
- [ ] @ita-social-projects/lv-734-03-streetcode 
- [ ] @ita-social-projects/lv-734-04-streetcode 

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Dobriyr — APPROVED** [review]

**Review by Tysyatsky — APPROVED** [review]
lgtm

File: Streetcode/Streetcode.WebApi/Extensions/ConfigurationBuilderExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿namespace Streetcode.WebApi.Extensions
+{
+    public static class ConfigurationBuilderExtensions
+    {
+        public static IConfigurationBuilder CustomConfigure(this IConfigurationBuilder builder, string environment)
```
Comment (OlehHatsko) [inline Streetcode/Streetcode.WebApi/Extensions/ConfigurationBuilderExtensions.cs:5]: Як на мене, то краще використовувати дієслово, як перше слово в назві метода.

**Review by OlehHatsko — APPROVED** [review]

File: Streetcode/Streetcode.XIntegrationTest/BlobServiceTests/FindFileTests/FindFileInStorageAsBase64Tests.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                Assert.Empty(result);
+            });
+        }
+
+        [Theory]
+        [InlineData("not-valid-file-name")]
+        public void ShouldThrowException_WhenInvalidFileName(string notValidFileName)
+        {
+            // Arrange
+            string result = "";
```
Comment (Katerix) [inline Streetcode/Streetcode.XIntegrationTest/BlobServiceTests/FindFileTests/FindFileInStorageAsBase64Tests.cs:59]: same

**Review by Katerix — COMMENTED** [review]
Well done! Please fix some code areas and merge pr

**Pull request merged**